### ISSUE TITLE
we have lost any way to know what size download a video is

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#23116]
when i hit click to play an inlined video, i no longer have any way to know if it's 5MB or 50MB. this sucks on roaming data.

**Comment by HarHarLinks** [comment]
this feature was removed in https://github.com/matrix-org/matrix-react-sdk/pull/6386, see also #18197 for more info

**Comment by ara4n** [comment]
yup, i know, and it's [a really frustrating oversight](https://github.com/vector-im/element-web/issues/18197#issuecomment-889513790)
